server/rematch/celery.py

```diff
@@ -6,6 +6,8 @@
 
 # set the default Django settings module for the 'celery' program.
 os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'rematch.settings')
+import django
+django.setup()
```

Comment:
I think you're already writing the tmux script that will include the right celery command.
We should perhaps add an "how to run" section to our readme later on.

To me the hackish `django.setup()` can be removed in favor of future PRs documenting how to execute the server components and specifically celery.

I'm using celery 4.0.0 on the mbp15 and an older version (3.*.*) on the mbp13 but we should hopefully support multiple versions of there's no critical issues between.